mods/ra/rules/defaults.yaml

```diff
@@ -491,6 +491,8 @@
 	Inherits@1: ^ExistsInWorld
 	Inherits@2: ^IronCurtainable
 	Inherits@3: ^SpriteActor
+	BodyOrientation:
+		CameraPitch: 255
```

Comment:
I forgot about the defenses when setting this, but it's easy to fix as all defenses use footprint offsets and therefore don't really need the 'top-down' pitch.